aviary/subsystems/propulsion/test/test_custom_engine_model.py

```diff
@@ -204,5 +207,119 @@ def test_custom_engine(self):
         assert_near_equal(float(prob.get_val('traj.cruise.rhs_all.y')), 4., tol)
 
 
+@use_tempdirs
+class TurbopropTest(unittest.TestCase):
+    def test_turboprop(self):
+        phase_info = {
+            'pre_mission': {
+                'include_takeoff': False,
+                'external_subsystems': [],
+                'optimize_mass': True,
+            },
+            'cruise': {
+                "subsystem_options": {"core_aerodynamics": {"method": "computed"}},
+                "user_options": {
+                    "optimize_mach": False,
+                    "optimize_altitude": False,
+                    "polynomial_control_order": 1,
+                    "num_segments": 2,
+                    "order": 3,
+                    "solve_for_distance": False,
+                    "initial_mach": (0.76, "unitless"),
+                    "final_mach": (0.76, "unitless"),
+                    "mach_bounds": ((0.7, 0.78), "unitless"),
+                    "initial_altitude": (35000.0, "ft"),
+                    "final_altitude": (35000.0, "ft"),
+                    "altitude_bounds": ((23000.0, 38000.0), "ft"),
+                    "throttle_enforcement": "boundary_constraint",
+                    "fix_initial": False,
+                    "constrain_final": False,
+                    "fix_duration": False,
+                    "initial_bounds": ((0.0, 0.0), "min"),
+                    "duration_bounds": ((30., 60.), "min"),
+                },
+                "initial_guesses": {"times": ([0, 30], "min")},
+            },
+            'post_mission': {
+                'include_landing': False,
+                'external_subsystems': [],
+            }
+        }
+
+        filename = get_path('models/engines/turboprop_4465hp.deck')
+        options = get_option_defaults()
+        options.set_val(Aircraft.Engine.DATA_FILE, filename)
+        options.set_val(Aircraft.Engine.NUM_ENGINES, 2)
+        options.set_val(Aircraft.Engine.SUBSONIC_FUEL_FLOW_SCALER, 1.0)
```

Comment:
I hope we're going to hide these options from the novice user because otherwise the example looks really complicated.

Reply:
Yeah, this is part of the stepping stone!